src/main/groovy/si/dlabs/gradle/extensions/FindbugsExtension.groovy

```diff
@@ -7,8 +7,8 @@ import org.gradle.api.tasks.Input
  */
 class FindbugsExtension extends PluginBaseExtension {
 
-    @Input String reportFile = "reports/checkstyle/checkstyle.xml"
     @Input String effort = "max"
     @Input String reportLevel = "low"
+    @Input String reportType = "html"
```

Comment:
it would be better if this would be an array, so you could have both html and xml reports